codecov.yaml

```diff
@@ -0,0 +1,13 @@
+codecov:
+  notify:
+    wait_for_ci: no
+
+coverage:
+  status:
+    project:
+      default:
+        # TODO
+        threshold: 1%
```

Comment:
We probably want a bit more slack in the threshold, i.e. IIRC we intend to maintain a 90% coverage ratio and based on the current ~97-98, I'd say 5% is reasonable.